.github/workflows/generate-docs.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
```

Comment:
The token is persisted in the clone's config, so it will be reused for git push

Reply:
Oh, that's cool.